docs/metrics/README.md

```diff
@@ -32,6 +32,7 @@ MyFruitCounter.Add(7, new("color", "red"), new("name", "apple"), new("taste", "s
   [`TagList`](https://learn.microsoft.com/dotnet/api/system.diagnostics.taglist?view=net-7.0#remarks)
   avoids allocating any memory for up to eight tags, thereby, reducing the
   pressure on GC to free up memory.
```

Comment:
I don't think you need to add another point for this. The point above already talks about using `TagList`.